sdk/search/search-documents/swagger/Service.md

```diff
@@ -341,3 +341,26 @@ directive:
   transform: >
     $["x-ms-client-name"] = "name";
 ```
+
+### Deprecations
+
+```yaml
+directive:
+  from: swagger-document
+  where: $.definitions[?(@['x-ms-discriminator-value'] == "#Microsoft.Skills.Text.EntityRecognitionSkill")]
+  transform: $.description += "\n\n@deprecated EntityRecognitionSkill has been deprecated. See \nhttps://learn.microsoft.com/en-us/azure/search/cognitive-search-skill-deprecated";
+```
+
+```yaml
+directive:
+  from: swagger-document
+  where: $.definitions[?(@['x-ms-discriminator-value'] == "#Microsoft.Skills.Text.SentimentSkill")]
+  transform: $.description += "\n\n@deprecated SentimentSkill has been deprecated. See \nhttps://learn.microsoft.com/en-us/azure/search/cognitive-search-skill-deprecated";
```

Comment:
Can you confirm if this loads up correctly in the IDE for the user? The link to migration docs. Shouldn't we add these in the refdocs as well?

Reply:
@KarishmaGhiya The deprecation does load in the IDE. Do you mean the generated docs? The deprecation is linked in the generated docs, but I'm not sure if you're referring to another set of docs.

Reply:
As per offline discussion, the generated skill is re-exported with the docs from the Autorest transform.